contributing/markup-format.md

```diff
@@ -389,6 +389,8 @@ Property      | Description
 Note that if you're using code inside the Markdown file,
 leave two empty lines after `{{ /codetabs }}` to turn off spell checking inside the block.
 
+The examples files are set up in [a specific file](../docs/data/remote-examples/language-examples/examples.yaml).
+If that file or any examples have changed, they need to be re-generated by running `npm run dev`.
```

Comment:
:lady_beetle: This doesn't seem to fit here. It's not about markup format nor about code tabs. Some code tabs use the examples, but it's not necessarily true nor do the examples have to be in code tabs.
I'd suggest clearer wording, but I'm not certain what the intention of this is supposed to be.